lib/turn.rb

```diff
@@ -0,0 +1,27 @@
+require 'pry'
+
+class Turn
+    attr_reader :card
+    attr_accessor :guess
+    def initialize(guess,card)
+        @guess = guess
+        @card = card
+    end
+
+    def correct?
```

Comment:
Just wanted to suggest another way this method could have been written for brevity:

```ruby
def correct?
  @card.answer.capitalize == @guess.capitalize #<=== Conditionals equate to true or false by default so no need to do an explicit if/else
end
```

This is purely another way of writing this method, and I just wanted to mention it for reference.